src/platform/silabs/efr32/BLEManagerImpl.cpp

```diff
@@ -458,6 +458,17 @@ CHIP_ERROR BLEManagerImpl::ConfigureAdvertisingData(void)
     advData[index++] = CHIP_ADV_DATA_TYPE_SERVICE_DATA;                                         // AD type : Service Data
     advData[index++] = ShortUUID_CHIPoBLEService[0];                                            // AD value
     advData[index++] = ShortUUID_CHIPoBLEService[1];
+
+#if CHIP_DEVICE_CONFIG_BLE_EXT_ADVERTISING
+    // Check for extended advertisement interval and redact VID/PID if past the initial period.
+    if (mFlags.Has(Flags::kExtAdvertisingEnabled))
+    {
+        mDeviceIdInfo.SetVendorId(0x0000);
+        mDeviceIdInfo.SetProductId(0x0000);
```

Comment:
maybe just `0` is clearer than the 0x0000 as it is shorter and looks more like a `clear`